Adapt Read Tutorial (#536)
NeurodataWithoutBorders/matnwb: README.md

For more information regarding the MatNWB API or any of the NWB Core types in MatNWB, visit the [MatNWB API Documentation pages](https://neurodatawithoutborders.github.io/matnwb/doc/index.html).

## Under the Hood

> NWB files are HDF5 files with data stored according to the Neurodata Without Borders: Neurophysiology (NWB:N) [schema](https://github.com/NeurodataWithoutBorders/nwb-schema/tree/dev/core). The schema is described in a set of YAML documents  which defines the various types and their attributes.

Reviewer: ```suggestion
NWB files are HDF5 files with data stored according to the Neurodata Without Borders (NWB) [schema](https://github.com/NeurodataWithoutBorders/nwb-schema/tree/dev/core). The schema is described in a set of YAML documents  which defines the various types and their attributes.
```

We dropped the ": Neurophysiology" part now

## Caveats

> The NWB:N schema is in a state of some evolution. This package assumes a certain set of rules are used to define the schema. As the schema is updated, some of the rules may be changed and these will break this package.

Reviewer: ```suggestion
The NWB schema has regular updates to add new data types or slightly modify existing data types. This package assumes a certain set of rules are used to define the schema. As the schema is updated, some of the rules may be changed and these will break this package.
```

For those planning on using matnwb alongside pynwb, please keep the following in mind:
 - MatNWB is dependent on the schema, which may not necessary correspond with your PyNWB schema version.  Please consider overwriting the contents within MatNWB's **~/schema/core** directory with the generating PyNWB's **src/pynwb/data directory** and running generateCore to ensure compatibilty between systems.
 
The `master` branch in this repository is considered perpetually unstable. If you desire Matnwb's full functionality (full round-trip with nwb data), please consider downloading the more stable releases in the Releases tab. Most releases will coincide with nwb-schema releases and guarantee compatibility of new features introduced with the schema release along with backwards compatibility with all previous nwb-schema releases.

> This package reads and writes NWB:N 2.0 files and does not support older formats.

Reviewer: ```suggestion
This package reads and writes NWB 2.0 files and does not support older formats.
```

## Examples

> [Basic File Reading](https://neurodatawithoutborders.github.io/matnwb/tutorials/html/read_demo.html) | a demo showcase for basic visualization from a DANDI dataset.

Reviewer: Can we move this to the Tutorials section, after "Intro to MatNWB"?
